src/declarations/stencil-public-compiler.ts

```diff
@@ -2192,30 +2192,268 @@ export type OutputTarget =
+   * skipped. For more information, see the definition of `strict` in the `glob`
+   * [documentation](https://github.com/isaacs/node-glob#options).
+   *
+   * @default true
+   */
+  globStrict?: boolean | undefined;
+
+  /**
+   * Any search parameter names that match against one of the RegExp in this array
+   * will be removed before looking for a precache match. This is useful if your
+   * users might request URLs that contain, for example, URL parameters used to
+   * track the source of the traffic.
+   *
+   * @default [/^utm_/]
+   */
+  ignoreURLParametersMatching?: RegExp[] | undefined;
+
+  /**
+   * A list of JavaScript files that should be passed to [`importScripts()`](https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/importScripts)
+   * inside the generated service worker file. This is  useful when you want to
+   * let Workbox create your top-level service worker file, but want to include
+   * some additional code, such as a push event listener.
+   */
+  importScripts?: string[] | undefined;
+
+  /**
+   * Whether the runtime code for the Workbox library should be included in the
+   * top-level service worker, or split into a separate file that needs to be deployed
+   * alongside the service worker. Keeping the runtime separate means that users will
+   * not have to re-download the Workbox code each time your top-level service worker changes.
+   *
+   * @default false
+   */
+  inlineWorkboxRuntime?: boolean | undefined;
+
+  /**
+   * One or more functions which will be applied sequentially against the
+   * generated manifest. If `modifyURLPrefix` or `dontCacheBustURLsMatching` are
+   * also specified, their corresponding transformations will be applied first.
+   */
+  manifestTransforms?: any[] | undefined;
+
+  /**
+   * This value can be used to determine the maximum size of files that will be
+   * precached. This prevents you from inadvertently precaching very large files
+   * that might have accidentally matched one of your patterns.
+   *
+   * @default 2097152
+   */
+  maximumFileSizeToCacheInBytes?: number | undefined;
+
+  /**
+   * If set to 'production', then an optimized service worker bundle that excludes
+   * debugging info will be produced. If not explicitly configured here, the `mode`
+   * value configured in the current `webpack` compiltion will be used.
+   */
+  mode?: string | undefined;
+
+  /**
+   * A mapping of prefixes that, if present in an entry in the precache manifest,
+   * will be replaced with the corresponding value. This can be used to, for example,
+   * remove or add a path prefix from a manifest entry if your web hosting setup
+   * doesn't match your local filesystem setup. As an alternative with more flexibility,
+   * you can use the `manifestTransforms` option and provide a function that modifies
+   * the entries in the manifest using whatever logic you provide.
+   */
+  modifyURLPrefix?:
```

Comment:
Similar to my comment above, it looks like we narrow this type from `any` to `{ [key: string]: string; }`, which we might want to consider a breaking change as well